src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -205,7 +205,7 @@ public SearchQueryResultModel querySnapshot(
         for (SearchHit hit : hits) {
             Map<String, String> hitsMap = new HashMap<>();
             for (Map.Entry<String, Object> entry : hit.getSourceAsMap().entrySet()) {
-                String key = entry.getKey();
+                String key = TimUtils.decode(entry.getKey());
                 String value = (String) entry.getValue();
```

Comment:
I know it's not this PR, but do you think it might be safer to do a toString in case the value isn't a string?  You'd just need to do a null check

Reply:
I'm not wild about this either, my thinking was to change the result model so we could use `Object` here. But I wouldn't change it to use `toString`. If the contents aren't a string, I'd rather get an exception so we can look into it.